source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4196,6 +4196,35 @@ export class Choice extends SerializableObject {
     }
 }
 
+export class ChoiceSetInputDataQuery extends SerializableObject {
+    //#region Schema
+
+    static readonly typeProperty = new StringProperty(
```

Comment:
@jwoo-msft Do we normally include the `type` property here?

Reply:
Type here needs to be defined since I am not extending from CardObject but SerializableObject. . The intention here is to not regard "choices.data" if the type is not "Data.Query".  I am not sure what is the best way to do it. Currently, I am checking the type and dataset to allow dynamic typeahead supported choiceset to be rendered. Other ways to handle could be doing something with internalparse or internalvalidateproperties method but I feel that is unnecessary. Let me know your thoughts. @jwoo-msft @anna-dingler